cg/services/order_validation_service/models/existing_case.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class ExistingCase(BaseModel):
+    internal_id: str
+    panels: list[str]
```

Comment:
To me it is really strange that we require the client to provide a list of panels when they use an existing case. Sure, they are allowed to provide a new set of panels, but it should not be required.

We decided to postpone that discussion though, so disregard this.

Reply:
For me, given that the frontend sends the old panels as default, I think sending `None` indicates an active effort by the customer to remove the panels. But let's postpone the discussion